doc/nrf/ug_matter_overview_dfu.rst

```diff
@@ -0,0 +1,189 @@
+.. _ug_matter_overview_dfu:
+
+Matter OTA
+##########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Matter Over-The-Air is a process that allows a Matter device in a Matter fabric to update its firmware.
+Most of the Matter devices will require such an update at some point during their lifecycle.
+The software image is provided by another Matter node, which acquired information about it and downloaded it from a centralized and reliable source.
+The download takes place over a special transfer protocol and user consent is required for the image to be applied on the target Matter device.
+
+.. _ug_matter_overview_dfu_roles:
+
+Matter OTA roles
+****************
+
+The following roles participate in the Matter OTA process:
+
+OTA Requestor
+   Any Matter device that is going to have its firmware updated.
```

Comment:
Please add information that OTA requestor is typically implemented on accessories (embedded devices - such as one built using nRF Connect SDK).